### PR TITLE
feat: add `rudof`

### PR DESCRIPTION
**Pull request by jeswr**
[rudof](https://rudof-project.github.io/rudof/cli_usage/convert.html) was recently presented by @labra at ISWC!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
 
 ## SHACL Validators
 
+- [rudof](rudof) [![Latest Version](https://img.shields.io/crates/v/rudof-cli.svg)](https://crates.io/crates/rudof-cli) [![PyPI](https://img.shields.io/pypi/v/pyrudof)](https://pypi.org/project/pyrudof/) [![rudof](https://github.com/rudof-project/rudof/actions/workflows/ci.yml/badge.svg)](https://github.com/rudof-project/rudof/actions/workflows/ci.yml) [![dependency status](https://deps.rs/repo/github/rudof-project/rudof/status.svg)](https://deps.rs/repo/github/rudof-project/rudof) is a library that implements [Shape Expressions](https://shex.io/), [SHACL](https://www.w3.org/TR/shacl/), [DCTAP](https://www.dublincore.org/specifications/dctap/) and other technologies in the [RDF](https://www.w3.org/RDF/) ecosystem. The library is implemented in [Rust](https://www.rust-lang.org/) and it also provides Python bindings.
```
Comment (TallTed) [inline README.md:32]:
```suggestion
- [rudof](rudof) [![Latest Version](https://img.shields.io/crates/v/rudof-cli.svg)](https://crates.io/crates/rudof-cli) [![PyPI](https://img.shields.io/pypi/v/pyrudof)](https://pypi.org/project/pyrudof/) [![rudof](https://github.com/rudof-project/rudof/actions/workflows/ci.yml/badge.svg)](https://github.com/rudof-project/rudof/actions/workflows/ci.yml) [![dependency status](https://deps.rs/repo/github/rudof-project/rudof/status.svg)](https://deps.rs/repo/github/rudof-project/rudof) is a library that implements [Shape Expressions](https://shex.io/), [SHACL](https://www.w3.org/TR/shacl/), [DCTAP](https://www.dublincore.org/specifications/dctap/), and other technologies in the [RDF](https://www.w3.org/RDF/) ecosystem. The library is implemented in [Rust](https://www.rust-lang.org/) and also provides Python bindings.
```

File: README.md [hunk 2]
```diff
@@ -48,6 +49,7 @@ Semantic shapes are frequently described using the SHACL or ShEx language.
 
 ## ShEx Validators
 
+- [rudof](rudof) [![Latest Version](https://img.shields.io/crates/v/rudof-cli.svg)](https://crates.io/crates/rudof-cli) [![PyPI](https://img.shields.io/pypi/v/pyrudof)](https://pypi.org/project/pyrudof/) [![rudof](https://github.com/rudof-project/rudof/actions/workflows/ci.yml/badge.svg)](https://github.com/rudof-project/rudof/actions/workflows/ci.yml) [![dependency status](https://deps.rs/repo/github/rudof-project/rudof/status.svg)](https://deps.rs/repo/github/rudof-project/rudof) is a library that implements [Shape Expressions](https://shex.io/), [SHACL](https://www.w3.org/TR/shacl/), [DCTAP](https://www.dublincore.org/specifications/dctap/) and other technologies in the [RDF](https://www.w3.org/RDF/) ecosystem. The library is implemented in [Rust](https://www.rust-lang.org/) and it also provides Python bindings.
```
Comment (TallTed) [inline README.md:52]:
```suggestion
- [rudof](rudof) [![Latest Version](https://img.shields.io/crates/v/rudof-cli.svg)](https://crates.io/crates/rudof-cli) [![PyPI](https://img.shields.io/pypi/v/pyrudof)](https://pypi.org/project/pyrudof/) [![rudof](https://github.com/rudof-project/rudof/actions/workflows/ci.yml/badge.svg)](https://github.com/rudof-project/rudof/actions/workflows/ci.yml) [![dependency status](https://deps.rs/repo/github/rudof-project/rudof/status.svg)](https://deps.rs/repo/github/rudof-project/rudof) is a library that implements [Shape Expressions](https://shex.io/), [SHACL](https://www.w3.org/TR/shacl/), [DCTAP](https://www.dublincore.org/specifications/dctap/), and other technologies in the [RDF](https://www.w3.org/RDF/) ecosystem. The library is implemented in [Rust](https://www.rust-lang.org/) and also provides Python bindings.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by jeswr** [comment]
Issues identified in https://github.com/w3c-cg/awesome-semantic-shapes/issues/15 should now be resolved. Please feel free to make any further changes directly.

**Comment by VladimirAlexiev** [comment]
@jeswr do we have these from https://rudof-project.github.io/rudof/#related-projects ?
- [ShEx-s](https://www.weso.es/shex-s/), a Scala implementation of ShEx (This project started as a re-implementation of ShEx-s in Rust).
- [SHACL-s](https://www.weso.es/shacl-s/), a Scala implementation of SHACL.
- [ShEx.js](https://github.com/shexjs/shex.js), a Javascript implementation of ShEx.

**Comment by jeswr** [comment]
> @jeswr do we have these from https://rudof-project.github.io/rudof/#related-projects ?
> - [ShEx-s](https://www.weso.es/shex-s/), a Scala implementation of ShEx (This project started as a re-implementation of ShEx-s in Rust).
> - [SHACL-s](https://www.weso.es/shacl-s/), a Scala implementation of SHACL.
> - [ShEx.js](https://github.com/shexjs/shex.js), a Javascript implementation of ShEx.

Ah no, it would be good if they were added - can I leave this to you to do?

**Comment by VladimirAlexiev** [comment]
@jeswr added to https://github.com/w3c-cg/awesome-semantic-shapes/issues/18

**Pull request merged**